### PR TITLE
Allow test framework to use cookies and authorization header

### PR DESCRIPTION
**Pull request by abhishekjain88**

### CODE REVIEW

**Comment by abhishekjain88** [comment]
@bargenilesh Verified that integration tests are working fine.

**Comment by abhishekjain88** [comment]
@bargenilesh Added comments.

File: modules/functional-test/src/main/java/com/intuit/wasabi/tests/library/APIServerConnector.java [hunk 1]
```diff
@@ -223,9 +227,10 @@ private RequestSpecification constructRequestSpec() {
         reqBuilder.setRelaxedHTTPSValidation();
         reqBuilder.log(LogDetail.ALL); // NIT use setting to control this
 
-        if (userName != null) {
+        if (!Strings.isNullOrEmpty(userName)) {
```
Comment (iizrailevsky) [inline modules/functional-test/src/main/java/com/intuit/wasabi/tests/library/APIServerConnector.java]: @abhishekjain88 Thanks for using isNullOrEmpty(...) in this case!

**Comment by bargenilesh** [comment]
@abhishekjain88 Looks good!!! :shipit:

**Pull request merged**